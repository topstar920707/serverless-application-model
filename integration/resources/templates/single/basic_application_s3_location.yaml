Resources:
  MyNestedApp:
    Type: AWS::Serverless::Application
    Properties:
      Location: ${templateurl}
