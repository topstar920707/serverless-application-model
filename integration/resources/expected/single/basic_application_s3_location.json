[
    { "LogicalResourceId":"MyNestedApp", "ResourceType":"AWS::CloudFormation::Stack" }
]
